lib/tasks/organisations.rake

```diff
@@ -51,10 +51,62 @@ namespace :organisations do
     org.name = new_name
     puts "Renamed org #{old_name} to #{new_name}" if org.save!
   end
+
+  desc "Merge an organisation into another"
+  task :merge, %i[source_organisation_slug target_organisation_slug] => :environment do |_, args|
```

Comment:
When running `organisations:merge`, the usage message doesn't include the task name - I think because the first argument should be `task`:

```suggestion
  task :merge, %i[source_organisation_slug target_organisation_slug] => :environment do |task, args|
```

Reply:
Ah, good spot, thanks! That should be fixed now.